test/test-functions

```diff
@@ -667,6 +667,8 @@ install_dbus() {
     # Fedora rawhide replaced dbus.service with dbus-daemon.service
     if [ -f $ROOTLIBDIR/system/dbus-daemon.service ]; then
         inst $ROOTLIBDIR/system/dbus-daemon.service
+        inst /usr/bin/dbus-broker
+        inst /usr/bin/dbus-broker-launch
```

Comment:
hmm, but there are systems that still use dbus-daemon, no? i.e. upgraded ones? this looks ike this makes the test fail on those, no?

Reply:
The switch to dbus-broker happened concurrently with the dbus-daemon rename, so I don't think that there are actual systems where this would fail. But indeed, those two things should not be conditionalized together.

I replaced this part with 'inst -o' calls (if we have dbus-broker, just install it).